src/motion/basic_target_follower.cpp

```diff
@@ -3,17 +3,23 @@
 #include <ros/exception.h>
 #include <string>
 
+#include <geometry_msgs/Twist.h>
+
 #include "constructors/TwistStamped.h"
 #include "constructors/Vector3.h"
 
 BasicTargetFollower::BasicTargetFollower(float kSpeed, float kAlt, float kYaw)
     : kSpeed_(kSpeed), kAlt_(kAlt), kYaw_(kYaw) {
-  std::string velocity_topic;
-  if (!ros::param::get("velocity_topic", velocity_topic)) {
-    throw ros::Exception("Must specify angular_position_topic parameter");
-  }
+  std::string velocity_topic = "/rexrov/cmd_vel";
+  // if (!ros::param::get("velocity_topic", velocity_topic)) {
```

Comment:
Should we just switch all the topics to remaps instead of parameters? That seems to be the normal practice